### ISSUE TITLE
tdf.groupby loses the TrajDataFrame's parameters

### ISSUE THREAD

**Issue by matteoboh** [scikit-mobility/scikit-mobility#197]
When using a .groupby on a TrajDataFrame, its parameters apparently are not inherited.

Code for reproducibility:

```python
import skmob
data_list = [[1, 39.984094, 116.319236, '2008-10-23 13:53:05'], [1, 39.984198, 116.319322, '2008-10-23 13:53:06'], [1, 39.984224, 116.319402, '2008-10-23 13:53:11'], [1, 39.984211, 116.319389, '2008-10-23 13:53:16']]
tdf = skmob.TrajDataFrame(data_list, user_id=0, latitude=1, longitude=2, datetime=3)

type(tdf.parameters)  ## this returns dict

# defining a toy function to apply after the groupby
def test(tdf):
    tdf['fake_col'] = tdf['lat'] + tdf['lng']
    return tdf

tdf_ = tdf.groupby('uid', as_index=False).apply(test)

type(tdf_.parameters)  ## this returns NoneType
```

Packages versions:
scikit-mobility v. 1.2.2
pandas v. 1.3.3

**Comment by jonpappalord** [comment]
Hi @matteoboh. We cannot reproduce this error. Indeed, we run the code on scikit-mobility 1.2.2. and it outputs a `TrajDataFrame` with a `parameters` attribute. Could you provide another example on which you get the error above?

**Comment by matteoboh** [comment]
Hi, I guess this is related to Issue #204 and, apparently, it has to do with pandas versions greater than 1.2.5 (?).
I attach the .yml file with the environment in which I still get these kinds of error with the tdf's `_crs` parameter, just in case it could help you..
[mobility.zip](https://github.com/scikit-mobility/scikit-mobility/files/7808676/mobility.zip)

**Comment by jonpappalord** [comment]
issue solved in version 1.2.3
